bottom-bar/src/main/java/com/roughike/bottombar/BottomBarBadge.java

```diff
@@ -42,9 +42,13 @@
      *
      * @param count the value this Badge should show.
      */
-    void setCount(int count) {
+    void setCount(int count, boolean formatBadge) {
         this.count = count;
-        setText(String.valueOf(count));
+        if (formatBadge && count > 1000) {
+            setText(String.format(getResources().getString(R.string.badge_format), count / 1000));
```

Comment:
`R.string.badge_format` can be defined as `%1$dK`, then you can use it without calling `String.format()` as follows:
`getResources().getString(R.string.badge_format), count / 1000)`.